Source/Csla.AspNetCore/Blazor/ApplicationContextManagerInMemory.cs

```diff
@@ -40,14 +41,18 @@ public class ApplicationContextManagerInMemory : IContextManager, IDisposable
     /// </summary>
     protected ActiveCircuitState ActiveCircuitState { get; }
 
+    private readonly HttpContext HttpContext;
+
     /// <summary>
     /// Creates an instance of the object, initializing it
     /// with the required IServiceProvider.
     /// </summary>
+    /// <param name="httpContextAccessor"></param>
     /// <param name="authenticationStateProvider">AuthenticationStateProvider service</param>
     /// <param name="activeCircuitState"></param>
-    public ApplicationContextManagerInMemory(AuthenticationStateProvider authenticationStateProvider, ActiveCircuitState activeCircuitState)
+    public ApplicationContextManagerInMemory(IHttpContextAccessor httpContextAccessor, AuthenticationStateProvider authenticationStateProvider, ActiveCircuitState activeCircuitState)
     {
+      HttpContext = httpContextAccessor.HttpContext;
```

Comment:
Is `IHttpContextAccessor` _always_ registered within the DI? Even in a real Blazor WASM scenario? That looks dangerous.

Reply:
It is required by CSLA for other reasons, so yes, it is always registered or the app will fail on startup.

None of this code is run on the wasm client - this is all aspnetcore Blazor server stuff.